<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogFilePathClass</class>
 <widget class="QWidget" name="LogFilePathClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>57</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogFilePath</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Log path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_EditFilePath">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_ButtonSelect">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="saveFile">
       <property name="text">
        <string>File name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileName"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_ButtonOpen">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
